popupwindow.js

```diff
@@ -0,0 +1,122 @@
+const exitBtn = document.getElementById("close-menu");
+const aboutBtn = document.getElementById("about-2");
+const PortfolioBtn = document.getElementById("portifolio-2");
+const contactBtn = document.getElementById("contact-2");
+const nav = document.querySelector("nav");
+const btn_button = document.querySelector(".btn");
+const exitPopover = document.getElementById("close-popover");
+
+btn_button.addEventListener("click", () => {
+  div.classList.add("popover-display");
+});
+mobileBtn.addEventListener("click", () => {
+  nav.classList.add("menu-btn");
+});
+
+function display() {
+  nav.classList.remove("menu-btn");
+}
+
+exitBtn.addEventListener("click", display);
+aboutBtn.addEventListener("click", display);
+contactBtn.addEventListener("click", display);
+
+function popover() {
+  div.classList.remove("popover-display");
+}
+exitPopover.addEventListener("click", popover);
+
+const dataPopover = [
+  {
+    heading: "Tonic",
+    image: "./imagsPop/tonic.png",
+    content: `Lorem Ipsum is simply dummy text of the printing 
+		and typesetting industry. Lorem Ipsum has been the industry's 
+		standard dummy text ever since the 1500s, when an unknown 
+		printer took a galley of type and scrambled it to make a type 
+		specimen book. It has survived not only five centuries, but 
+		also the leap into electronic typesetting, remaining essent`,
+  },
+
+  {
+    heading: "Multi-post <br/>Stories",
+    image: "./imagsPop/multi1.png",
+    content: `Lorem Ipsum is simply dummy text of the printing 
+		and typesetting industry. Lorem Ipsum has been the industry's 
+		standard dummy text ever since the 1500s, when an unknown 
+		printer took a galley of type and scrambled it to make a type 
+		specimen book. It has survived not only five centuries, but 
+		also the leap into electronic typesetting, remaining essent`,
+  },
+
+  {
+    heading: "Facebook",
+    image: "./imagsPop/facebook1.png",
+    content: `Lorem Ipsum is simply dummy text of the printing 
+		and typesetting industry. Lorem Ipsum has been the industry's 
+		standard dummy text ever since the 1500s, when an unknown 
+		printer took a galley of type and scrambled it to make a type 
+		specimen book. It has survived not only five centuries, but 
+		also the leap into electronic typesetting, remaining essent`,
+  },
+
+  {
+    heading: "Multi-post <br/>Stories",
+    image: "./imagsPop/project image 2.png",
+    content: `Lorem Ipsum is simply dummy text of the printing 
+		and typesetting industry. Lorem Ipsum has been the industry's 
+		standard dummy text ever since the 1500s, when an unknown 
+		printer took a galley of type and scrambled it to make a type 
+		specimen book. It has survived not only five centuries, but 
+		also the leap into electronic typesetting, remaining essent`,
+  },
+];
+
+const modal = document.querySelector(".modal");
+const popupWrap = document.querySelector(".pop-up-wrap");
+const projectBtn = document.querySelectorAll(".see-btn");
+
```

Comment:
- This is what the project requirements ask for:
![image](https://user-images.githubusercontent.com/83596135/189400760-cec4f52c-ab6e-4962-b57a-e6f1f1ee0b49.png)

- [OPTIONAL] In this particular project, the section of the whole project along with the models should be created dynamically with Javascript. I can see that Modal popup data is populated with the data from the `dataPopover` array good job! 👏🏻 , but the base projects section is still not being created with Javascript. kindly make sure all the **[following code lines](https://github.com/Anwaar12/My-portfolio/blob/1f290bb57a6a2afcb06fcbdeda19ea669e3739dd/index.html#L64-L185)** is removed from your `index.html` file. The projects section and the model popup windows should be created dynamically with the information from the **projects array** you created in the js file(**i.e. the projects list should be removed, and instead it should be added via JavaScript**.) Here is an **[example](https://github.com/olawale-o/olawale-o.me/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95)**

- That way, adding projects in the future will be easy. You will only need to add it in one place - as another object in the ` projectsArray` array. 😉 👍🏻 
 
Your code in the HTML file should look like this:
```
      <section id="worksNew" class="worksNew">
          "insertion of work cards using JavaScript and DOM manipulation"
       </section>
```


Please take the time to read the following documentation like [this one](https://developer.mozilla.org/en-US/docs/Web/API/Document/createElement) or [this one](https://developer.mozilla.org/en-US/docs/Web/API/Element/insertAdjacentHTML).

- That way, adding projects in the future will be easy. You will only need to add it in one place - as another object in the `dataPopover` array. 😉 👍🏻 

### These are only optional because due to oversight the previous reviewer probably missed them. However, they are important. I strongy recommand you to make this 👍🏻